packages/frontend/core/src/blocksuite/presets/ai/chat-panel/chat-panel-input.ts

```diff
@@ -594,19 +632,13 @@ export class ChatPanelInput extends SignalWatcher(WithDisposable(LitElement)) {
 
         this.updateContext({ status: 'success' });
 
-        if (!this.chatContextValue.chatSessionId) {
-          this.updateContext({
-            chatSessionId: AIProvider.LAST_ROOT_SESSION_ID,
-          });
-        }
-
```

Comment:
@donteatfriedrice Can I remove this `LAST_ROOT_SESSION_ID` update? Don't know what "LAST_ACTION_SESSIONID" and "LAST_ROOT_SESSION_ID" are used for.